ossdbtoolsservice/parsers/datatype_parsers.py

```diff
@@ -65,7 +67,9 @@ def parse_time_with_timezone(value: str) -> datetime.time:
 
 
 def parse_datetime(value: str) -> datetime.datetime:
-    if value == 'now()':
+    if value == 'now()':  # PG
+        return datetime.datetime.now()
+    elif value == 'CURRENT_TIMESTAMP':  # MySQL
```

Comment:
With this change both values with work for both providers.
Can we check if it is requried?

Reply:
modified